### PR TITLE
Social media

### PR DESCRIPTION
**Pull request by bc-adamferenzi**
Ability to position social media icons in top nav, along with options to position left/right/none in footer.

### CODE REVIEW

**Comment by bc-adamferenzi** [comment]
for https://jira.bigcommerce.com/browse/MERC-1162

**Comment by bc-julianleong** [comment]
@bc-miko-ademagic

File: assets/scss/components/stencil/navUser/_navUser.scss [hunk 1]
```diff
@@ -123,6 +123,27 @@
...
+
+    @include breakpoint("medium") {
+        display: block;
+        margin-top: 10px;
+        padding-right: 10px;
+    }
+}
+
+span.navUser-item-social-break {
+    display: none;
```
Comment (bc-miko-ademagic) [inline assets/scss/components/stencil/navUser/_navUser.scss]: Can you write this statement without targeting a `span` specifically? We try not to style by tag, this is a bit too specific.

File: templates/components/common/navigation.html [hunk 2]
```diff
@@ -18,6 +18,14 @@
         <li class="navUser-item">
             <a class="navUser-action navUser-item--compare" href="{{urls.compare}}" data-compare-nav>{{lang 'common.compare'}} <span class="countPill countPill--positive countPill--alt"></span></a>
         </li>
+        {{#if theme_settings.social_icon_placement_top}}
+             <li class="navUser-item">   
+             <span class="navUser-item-social"> 
+                {{> components/common/social-links}} 
+             </span>   
+            </li>
+             <li class="navUser-item"> <span class="navUser-item-social-break">    |</span></li>
```
Comment (bc-miko-ademagic) [inline templates/components/common/navigation.html]: You should be able to solve this with CSS only. Is there a reason we're using new markup?

**Comment by mjschock** [comment]
closing as it's been reopened as #887

**Pull request closed without merging**